lib/public_key/src/public_key.erl

```diff
@@ -1700,9 +1689,12 @@ otp_cert(Der) when is_binary(Der) ->
 otp_cert(#'OTPCertificate'{} = Cert) ->
     Cert;
 otp_cert(#cert{otp = OtpCert}) ->
-    OtpCert;
-otp_cert(#'Certificate'{} = Cert) ->
-    pkix_decode_cert(der_encode('Certificate', Cert), otp).
```

Comment:
Why remove this conversion?

Reply:
I guess since most public_key functions take a der_cert() or otp_cert() and not a plain_cert()!?

Reply:
because ... I think ... this clause was added for initial OCSP implementation, which I changed in this PR.
removed clause is dropping the DER representation which I need at some later OCSP processing occasion.

In order to get DER later I would need to run similar conversion one more time, which I think is a waste.
For that reason, I created `combined_cert` function which does not loose partial processing result.

removed clause, became not useful.